### ISSUE TITLE
.NET 6 has hit EOL

### ISSUE THREAD

**Issue by GGG-KILLER** [KnossosNET/Knossos.NET#259]
.NET 6 has hit EOL on November 12, 2024[^1] and will no longer be supported nor receive updates.
Recommended path is to upgrade either to .NET 8 (LTS) or .NET 9 (STS).

We've hit this while removing .NET 6 and 7 from nixpgks: https://github.com/NixOS/nixpkgs/issues/326335.

[^1]: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

**Comment by Shivansps** [comment]
Yeah, we are aware of that, the problem on our case is that .NET 8 removes support for Windows 7 (that FSO still officially supports) and also increases the minimal OS version for MacOS.
So the plan is to continue development until the app is feature complete, that would be 1.4 hopefully, and then move to .NET8 while also providing a legacy OS version on .NET 6.
